keras_cv/layers/object_detection/roi_pool_test.py

```diff
@@ -0,0 +1,161 @@
+        self.assertAllClose(expected_feature_map, pooled_feature_map)
+
+    def test_roi_quantize_y(self):
+        roi_pooler = ROIPooler("yxyx", target_size=[2, 2], image_shape=[224, 224, 3])
+        feature_map = tf.expand_dims(tf.reshape(tf.range(64), [8, 8, 1]), axis=0)
+        rois = tf.reshape(tf.constant([0.0, 0.0, 224, 220]), [1, 1, 4])
+        pooled_feature_map = roi_pooler(feature_map, rois)
+        # the maximum value would be at bottom-right at each block, roi sharded into 2x2 blocks
+        # | 0, 1, 2             | 3, 4, 5, 6            |
+        # | 8, 9, 10            | 11, 12, 13, 14        |
+        # | 16, 17, 18          | 20, 21, 22, 23        |
+        # | 24, 25, 26(max)     | 27, 28, 29, 30(max)   |
+        # --------------------------------------------
+        # | 32, 33, 34          | 35, 36, 37, 38        |
+        # | 40, 41, 42          | 43, 44, 45, 46        |
+        # | 48, 49, 50          | 51, 52, 53, 54        |
+        # | 56, 57, 58(max)     | 59, 60, 61, 62(max)   |
+        # --------------------------------------------
+        expected_feature_map = tf.reshape(tf.constant([26, 30, 58, 62]), [1, 2, 2, 1])
+        self.assertAllClose(expected_feature_map, pooled_feature_map)
+
+    def test_roi_quantize_x(self):
+        roi_pooler = ROIPooler("yxyx", target_size=[2, 2], image_shape=[224, 224, 3])
+        feature_map = tf.expand_dims(tf.reshape(tf.range(64), [8, 8, 1]), axis=0)
+        rois = tf.reshape(tf.constant([0.0, 0.0, 220, 224]), [1, 1, 4])
+        pooled_feature_map = roi_pooler(feature_map, rois)
+        # the maximum value would be at bottom-right at each block, roi sharded into 2x2 blocks
+        # | 0, 1, 2, 3          | 4, 5, 6, 7            |
+        # | 8, 9, 10, 11        | 12, 13, 14, 15        |
+        # | 16, 17, 18, 19(max) | 20, 21, 22, 23(max)   |
+        # --------------------------------------------
+        # | 24, 25, 26, 27      | 28, 29, 30, 31        |
+        # | 32, 33, 34, 35      | 36, 37, 38, 39        |
+        # | 40, 41, 42, 43      | 44, 45, 46, 47        |
+        # | 48, 49, 50, 51(max) | 52, 53, 54, 55(max)   |
+        # --------------------------------------------
+        expected_feature_map = tf.reshape(tf.constant([19, 23, 51, 55]), [1, 2, 2, 1])
+        self.assertAllClose(expected_feature_map, pooled_feature_map)
+
+    def test_roi_quantize_h(self):
+        roi_pooler = ROIPooler("yxyx", target_size=[3, 2], image_shape=[224, 224, 3])
+        feature_map = tf.expand_dims(tf.reshape(tf.range(64), [8, 8, 1]), axis=0)
+        rois = tf.reshape(tf.constant([0.0, 0.0, 224, 224]), [1, 1, 4])
+        pooled_feature_map = roi_pooler(feature_map, rois)
+        # the maximum value would be at bottom-right at each block, roi sharded into 3x2 blocks
+        # | 0, 1, 2, 3          | 4, 5, 6, 7            |
+        # | 8, 9, 10, 11        | 12, 13, 14, 15        |
```

Comment:
missing max annotation for 11 and 15 here.

Reply:
Done.